majordomo_grpc.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
-// - protoc-gen-go-grpc v1.3.0
+// - protoc-gen-go-grpc v1.2.0
```

Comment:
Can you upgrade your `protoc-gen-go-grpc` to at least v1.3.0

Reply:
I've added Makefile checks for this purpose.